app/uk/gov/hmrc/exports/mongo/ExportsClient.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.mongo
+
+import com.mongodb.client.{MongoClient, MongoClients}
+import uk.gov.hmrc.exports.config.AppConfig
+
+trait ExportsClient {
```

Comment:
Is this trait really useful? It will be only used by `MigrationRoutine`, which is already at the top of the Migration package. It won't be used by any other class.